frontends/web/src/routes/account/account.tsx

```diff
@@ -179,16 +169,17 @@ class Account extends Component<Props, State> {
                     .catch(console.error);
                 }
             }
-            this.setState(state, this.onAccountChanged);
+            this.setState({ status }, this.onAccountChanged);
         })
         .catch(console.error);
     }
 
     private onAccountChanged = () => {
-        if (!this.props.code || this.state.fatalError) {
+        const status = this.state.status;
+        if (!this.props.code || status === undefined || status.fatalError) {
             return;
         }
-        if (this.state.accountSynced && !this.state.offlineMode) {
+        if (status.synced && !status.offline) {
```

Comment:
I'd assume it to be online if `status.synced` is true, is `!status.offline` really necessary?

Reply:
These are independent. You can lose the connection and reconnect after you have synced, for example.